src/forms/formHelpers.js

```diff
@@ -325,26 +322,26 @@ const ColumnHeading = function ( props ) {
 * 
 * @function
 * @param {object} props
-* @property {object} props.__ - explanation
+* @property {object} __ - explanation
```

Comment:
Just because it's destructured doesn't mean it's not a property of `props`, so this should probably stay the same.